src/modules/common/examples/multiplemovies.json

```diff
@@ -1,3324 +1,3298 @@
-			},
-			"rottenTomatoes": {
-				"votes": 0,
-				"value": 75,
-				"type": "user"
-			}
-		},
-		"movieFile": {
-			"movieId": 157,
-			"relativePath": "Lightyear (2022) WEBRip-1080p.mkv",
-			"path": "/movies/Lightyear (2022)/Lightyear (2022) WEBRip-1080p.mkv",
-			"size": 4162236849,
-			"dateAdded": "2022-07-14T12:39:18Z",
-			"sceneName": "Lightyear.2022.1080p.WEBRip.DD5.1.x264-EVO[TGx]",
-			"indexerFlags": 1,
-			"quality": {
-				"quality": {
-					"id": 15,
-					"name": "WEBRip-1080p",
-					"source": "webrip",
-					"resolution": 1080,
-					"modifier": "none"
-				},
-				"revision": {
-					"version": 1,
-					"real": 0,
-					"isRepack": false
-				}
-			},
-			"mediaInfo": {
-				"audioBitrate": 384000,
-				"audioChannels": 5.1,
-				"audioCodec": "AC3",
-				"audioLanguages": "eng",
-				"audioStreamCount": 1,
-				"videoBitDepth": 8,
-				"videoBitrate": 0,
-				"videoCodec": "x264",
-				"videoDynamicRangeType": "",
-				"videoFps": 23.976,
-				"resolution": "1920x804",
-				"runTime": "1:45:04",
-				"scanType": "Progressive",
-				"subtitles": ""
-			},
-			"originalFilePath": "Lightyear.2022.1080p.WEBRip.DD5.1.x264-EVO[TGx]/Lightyear.2022.1080p.WEBRip.DD5.1.x264-EVO.mkv",
-			"qualityCutoffNotMet": false,
-			"languages": [
-				{
-					"id": 1,
-					"name": "English"
-				}
-			],
-			"releaseGroup": "EVO",
-			"edition": "",
-			"id": 227
-		},
-		"id": 157
-	},
-	{
-		"title": "Minions: The Rise of Gru",
-		"originalTitle": "Minions: The Rise of Gru",
-		"originalLanguage": {
-			"id": 1,
-			"name": "English"
-		},
-		"alternateTitles": [
-			{
-				"sourceType": "tmdb",
-				"movieId": 165,
-				"title": "미니언즈: 라이즈 오브 그루",
-				"sourceId": 0,
-				"votes": 0,
-				"voteCount": 0,
-				"language": {
-					"id": 1,
-					"name": "English"
-				},
-				"id": 1679
-			},
-			{
-				"sourceType": "tmdb",
-				"movieId": 165,
-				"title": "Miljenici: Uspon Grua",
-				"sourceId": 0,
-				"votes": 0,
-				"voteCount": 0,
-				"language": {
-					"id": 1,
-					"name": "English"
-				},
-				"id": 1680
-			},
-			{
-				"sourceType": "tmdb",
-				"movieId": 165,
-				"title": "Käsilased 2",
-				"sourceId": 0,
-				"votes": 0,
-				"voteCount": 0,
-				"language": {
-					"id": 1,
-					"name": "English"
-				},
-				"id": 1681
-			},
-			{
-				"sourceType": "tmdb",
-				"movieId": 165,
-				"title": "Minions 2: Η Άνοδος του Γκρου",
-				"sourceId": 0,
-				"votes": 0,
-				"voteCount": 0,
-				"language": {
-					"id": 1,
-					"name": "English"
-				},
-				"id": 1682
-			},
-			{
-				"sourceType": "tmdb",
-				"movieId": 165,
-				"title": "Minioni 2",
-				"sourceId": 0,
-				"votes": 0,
-				"voteCount": 0,
-				"language": {
-					"id": 1,
-					"name": "English"
-				},
-				"id": 1683
-			},
-			{
-				"sourceType": "tmdb",
-				"movieId": 165,
-				"title": "Minions: Historien om Gru",
-				"sourceId": 0,
-				"votes": 0,
-				"voteCount": 0,
-				"language": {
-					"id": 15,
-					"name": "Norwegian"
-				},
```

Comment:
@ajnart can we remove this *huge* file and use a unit test instead?

Reply:
Yes of course 👍🏼  But i'll close this PR for now